main.js

```diff
@@ -0,0 +1,436 @@
+// TO DO
+// reset gameboard
+// add css disc movement elements
+// reconfigure redundant code
+// what is JQuery and API and why would it help me display the game in HTML
+// write a test in Jasmine
+// give options to change of numbers pegs and discs
+// add function to automatically complete the game
+
+let towerOne = [5, 4, 3, 2, 1];
+let towerTwo = [];
+let towerThree = [];
+
+let board = [towerOne, towerTwo, towerThree];
```

Comment:
I see you nested them here, but you could remove this and just have on variable to hold everything for the board